app/controllers/ems_block_storage_controller.rb

```diff
@@ -0,0 +1,43 @@
+class EmsBlockStorageController < ApplicationController
```

Comment:
Never mind that - @h-kataria has explained it to me.  I have made the required changes to each controller represented by a modified entry in the Menu::DefaultMenu.